delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/MainSuite.scala

```diff
@@ -42,7 +42,7 @@ class MainSuite extends CatsEffectSuite with MainSuite.Fixture {
       make[ContextShift[IO]].fromValue(contextShift)
       make[Timer[IO]].fromValue(timer)
     }
-    val (cfg, config, cl, pDefs) = Main.loadPluginsAndConfig(pluginLoaderConfig).unsafeRunSync()
+    val (cfg, config, cl, pDefs) = Main.loadPluginsAndConfig(pluginLoaderConfig).accepted
```

Comment:
Whenever we can, we should avoid to block in MUnit tests

Reply:
That's ok, but I will not try and fix all these instances in this PR (I switched to accepted here just so that the timeout will be effective)